homeassistant/components/bluesound/media_player.py

```diff
@@ -449,6 +448,23 @@ async def async_update_status(self):
             _LOGGER.info("Client connection error, marking %s as offline", self._name)
             raise
 
+    @property
+    def unique_id(self):
+        """Return an unique ID."""
+        return f"{format_mac(self._sync_status['@mac'])}-{self.port}"
+
+    @property
+    def device_info(self):
```

Comment:
There's no config flow or config entries. Then device info won't be used.

https://developers.home-assistant.io/docs/device_registry_index#automatic-registration-through-an-entity

Reply:
My intention is to build config flow next week and you usually prefer small prs..

Reply:
Then add the device info in a third PR. Thanks!